reconstruction/recoiltof/pom.xml

```
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+ <project xmlns="http://maven.apache.org/POM/4.0.0"
+ xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.jlab.clas12.detector</groupId>
+  <artifactId>clas12detector-recoiltof</artifactId>
+  <version>1.0-SNAPSHOT</version>
```

match the current coatjava version
```suggestion
  <version>13.4.0-SNAPSHOT</version>
```